addons/mail/static/src/models/activity_mark_done_popover_view/activity_mark_done_popover_view.js

```diff
@@ -37,10 +37,11 @@ registerModel({
         /**
          * Handles click on this "Done & Schedule Next" button.
          */
-        onClickDoneAndScheduleNext() {
-            this.activityViewOwner.activity.markAsDoneAndScheduleNext({
+        async onClickDoneAndScheduleNext() {
+            await this.activityViewOwner.activity.markAsDoneAndScheduleNext({
                 feedback: this.feedbackTextareaRef.el.value,
             });
+            this.component.trigger('reload', { keepChanges: true });
```

Comment:
But here at minimum, as the method is async, you'll need to make sure `this.component` (still) exists before calling trigger on it.